recipes/fastfilters2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "fastfilters2" %}
+{% set version = "2.0.0.dev0" %}
+{% set sha256 = "a671418a47079f165c18d4e8ef0415fa10dbfd4d7b458a384e856781864ed010" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install -vv .
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
```

Comment:
```suggestion
    - {{ compiler("cxx") }}
    - make
```

Reply:
Specified `cmake` and `ninja` (for non-Windows platforms), [as suggested by scikit-build](https://scikit-build.readthedocs.io/en/latest/usage.html#example-of-setup-py-cmakelists-txt-and-pyproject-toml).